js/src/views/ParityBar/accountStore.js

```diff
@@ -111,10 +111,16 @@ export default class AccountStore {
   }
 
   subscribeDefaultAccount () {
-    return this._api.subscribe('parity_defaultAccount', (error, defaultAccount) => {
+    this._api.subscribe('parity_defaultAccount', (error, defaultAccount) => {
```

Comment:
There is a reason for the return - when testing, you can call the function and do the expectations after promise has resolved. (This way you don't end up with unresolved promises or possible fragile wait-states.)

So revert and rather put the other subscription explicitly in an own function.